apps/yapms/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -65,6 +66,36 @@ function findCandidate(id: string | null) {
 	}
 }
 
+function getCandidatesForRegion(candidateStr: string | null, value: number) {
+	if (candidateStr === null) {
+		return [{ candidate: get(TossupCandidateStore), count: value, margin: 0 }];
+	}
+	try {
+		const parsedArr: RegionCandidates = JSON.parse(candidateStr);
+		RegionCandidatesSchema.parse(parsedArr);
+		let totCount = 0;
+		const candidateArr = [];
+		parsedArr.forEach((elem) => {
+			totCount += elem.count;
+			candidateArr.push({
+				...elem,
+				candidate: findCandidate(elem.candidate)
+			});
+		});
+		if (totCount !== value) {
+			candidateArr.push({
+				candidate: get(TossupCandidateStore),
+				count: value - totCount,
+				margin: 0
+			});
+		}
```

Comment:
This will cause the tossup count to go negative. Would it be better to just return the tossup candidate and console log an error?

![image](https://github.com/yapms/yapms/assets/13600696/b553a905-8a33-4d2e-9892-eca940b8633f)